js/__internal/grids/grid_core/rows_view/module.ts

```diff
@@ -1,3 +1,6 @@
+/* eslint-disable @typescript-eslint/prefer-nullish-coalescing */
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
Why do we need unused vars here?
Maybe we can remove them from the methods?

Reply:
It's for propery typing. Variable isn't used in method, but used in method override